packages/react-divider/MIGRATION.md

```diff
@@ -0,0 +1,60 @@
+# Divider Migration
+
+## STATUS: WIP 🚧
+
+This Migration guide is a work in progress and is not yet ready for use.
+
+## Migration from v8
+
+The existing `Separator` control supports the `content` property as well as children nodes. The `content` property has been removed and now only supports children.
+
+With the new converged capabilities, both the `styles` and `theme` properties are no longer supported.
+
+Properties that are still available are the `alignContent` and `vertical` with the same acceptable values that are currently defined.
+
+## Migration from v0
+
+The v0 Divider is close to the converged Divider. Again, as with the Separator control, the `content` property in the converged component is not supported.
+
+Also, from design discussion, the converged component does not support the `size` component as it is redundant as it can be replicated through using the `borderSize` property and if needed, update the `fontSize` property to match desired style.
+
+The `fitted` property also is not in use as by default the `Divider` can use the standard `margin` | `marginLeft` | `marginRight` | `marginTop` | `marginBottom` properties to adjust as needed. The default margin for the divider is 0.
```

Comment:
Isn't `inset` the replacement for `fitted` (except the opposite meaning)? 
So for v0 => converged:
`<Divider fitted />` => `<Divider />`
`<Divider />` => `<Divider inset />`

Reply:
There is no longer a fitted as fitted only modified margins. While technically you could use fitted to emulate the inset, inset uses a specified padding amount (not margins). This ensures that the whole block still fills the appropriate space in rendering